Cannot log into 2 devices simultaneously?

**Description**
Using both iPad and laptop; iPad able to load GATool and announce screens. Laptop opens GATool but shows a blank screen beyond event selection screen. Attempted multiple browsers, logout/relogin on laptop

**Workarounds**
Just use iPad

**Expected behavior**
Both tools should be able to used

**Environment**
* Browser type and version: On iPad, Safari. On laptop: Chrome, Sfari

I cleared the browser cache and logged in again. Same issue, here is what the error was:

The FetchEvent for "<URL>" resulted in a network error response: the promise was rejected.
The FetchEvent for "<URL>" resulted in a network error response: the promise was rejected.
The FetchEvent for "<URL>" resulted in a network error response: the promise was rejected.
The FetchEvent for "<URL>" resulted in a network error response: the promise was rejected.
The FetchEvent for "<URL>" resulted in a network error response: the promise was rejected.
The FetchEvent for "<URL>" resulted in a network error response: the promise was rejected.
service-worker.js:1 Uncaught (in promise) 
_handle @ service-worker.js:1
service-worker.js:1 Uncaught (in promise) 
service-worker.js:1 Uncaught (in promise) 
service-worker.js:1 Uncaught (in promise) 
service-worker.js:1 Uncaught (in promise) 
service-worker.js:1 Uncaught (in promise) 
newrelic.js:1  Failed to load resource: net::ERR_FAILED
p.css:1  Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
 Error: <svg> attribute width: Unexpected end of attribute. Expected length, "".
y @ main.84114ffc.js:2
 Event refresh stopped at App level.
App.jsx:2159 Team list changed. Fetching Community Updates.
App.jsx:1333 Fetching community updates for Las Vegas Regional...
App.jsx:1981 starting to load event
App.jsx:1983 Conditions match to load Las Vegas Regional...
App.jsx:2293 Fetching robot images for Las Vegas Regional...
App.jsx:803 Fetching team list for Las Vegas Regional...
App.jsx:487 Fetching schedule for Las Vegas Regional...
App.jsx:518 Fetching Practice Schedule for Las Vegas Regional...
react-dom.production.min.js:24 Error: <svg> attribute width: Unexpected end of attribute. Expected length, "".
y @ react-dom.production.min.js:24
react-dom.production.min.js:24 Error: <svg> attribute width: Unexpected end of attribute. Expected length, "".
y @ react-dom.production.min.js:24
App.jsx:554 Fetching Qual Schedule for Las Vegas Regional...
App.jsx:2159 Team list changed. Fetching Community Updates.
App.jsx:1333 Fetching community updates for Las Vegas Regional...
App.jsx:2293 Fetching robot images for Las Vegas Regional...
App.jsx:621 It's after matches start. Resetting Practice Schedule
App.jsx:640 Fetching Playoff Schedule for Las Vegas Regional...
App.jsx:1476 Fetching Ranks for Las Vegas Regional...
react-dom.production.min.js:189 TypeError: Cannot read properties of undefined (reading '8022')
    at AnnouncePage.jsx:39:40
    at AnnouncePage.jsx:166:36
    at Array.forEach (<anonymous>)
    at UW (AnnouncePage.jsx:165:22)
    at Ca (react-dom.production.min.js:167:137)
    at Cs (react-dom.production.min.js:290:337)
    at wc (react-dom.production.min.js:280:389)
    at bc (react-dom.production.min.js:280:320)
    at gc (react-dom.production.min.js:280:180)
    at ac (react-dom.production.min.js:271:88)
fl @ react-dom.production.min.js:189
react-dom.production.min.js:127 Uncaught TypeError: Cannot read properties of undefined (reading '8022')
    at AnnouncePage.jsx:39:40
    at AnnouncePage.jsx:166:36
    at Array.forEach (<anonymous>)
    at UW (AnnouncePage.jsx:165:22)
    at Ca (react-dom.production.min.js:167:137)
    at Cs (react-dom.production.min.js:290:337)
    at wc (react-dom.production.min.js:280:389)
    at bc (react-dom.production.min.js:280:320)
    at gc (react-dom.production.min.js:280:180)
    at ac (react-dom.production.min.js:271:88)

> I cleared the browser cache and logged in again. Same issue, here is what the error was:
> 
> The FetchEvent for "" resulted in a network error response: the promise was rejected. The FetchEvent for "" resulted in a network error response: the promise was rejected. The FetchEvent for "" resulted in a network error response: the promise was rejected. The FetchEvent for "" resulted in a network error response: the promise was rejected. The FetchEvent for "" resulted in a network error response: the promise was rejected. The FetchEvent for "" resulted in a network error response: the promise was rejected. service-worker.js:1 Uncaught (in promise) _handle @ service-worker.js:1 service-worker.js:1 Uncaught (in promise) service-worker.js:1 Uncaught (in promise) service-worker.js:1 Uncaught (in promise) service-worker.js:1 Uncaught (in promise) service-worker.js:1 Uncaught (in promise) newrelic.js:1 Failed to load resource: net::ERR_FAILED p.css:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT Error: attribute width: Unexpected end of attribute. Expected length, "". y @ main.84114ffc.js:2 Event refresh stopped at App level. App.jsx:2159 Team list changed. Fetching Community Updates. App.jsx:1333 Fetching community updates for Las Vegas Regional... App.jsx:1981 starting to load event App.jsx:1983 Conditions match to load Las Vegas Regional... App.jsx:2293 Fetching robot images for Las Vegas Regional... App.jsx:803 Fetching team list for Las Vegas Regional... App.jsx:487 Fetching schedule for Las Vegas Regional... App.jsx:518 Fetching Practice Schedule for Las Vegas Regional... react-dom.production.min.js:24 Error: attribute width: Unexpected end of attribute. Expected length, "". y @ react-dom.production.min.js:24 react-dom.production.min.js:24 Error: attribute width: Unexpected end of attribute. Expected length, "". y @ react-dom.production.min.js:24 App.jsx:554 Fetching Qual Schedule for Las Vegas Regional... App.jsx:2159 Team list changed. Fetching Community Updates. App.jsx:1333 Fetching community updates for Las Vegas Regional... App.jsx:2293 Fetching robot images for Las Vegas Regional... App.jsx:621 It's after matches start. Resetting Practice Schedule App.jsx:640 Fetching Playoff Schedule for Las Vegas Regional... App.jsx:1476 Fetching Ranks for Las Vegas Regional... react-dom.production.min.js:189 TypeError: Cannot read properties of undefined (reading '8022') at AnnouncePage.jsx:39:40 at AnnouncePage.jsx:166:36 at Array.forEach () at UW (AnnouncePage.jsx:165:22) at Ca (react-dom.production.min.js:167:137) at Cs (react-dom.production.min.js:290:337) at wc (react-dom.production.min.js:280:389) at bc (react-dom.production.min.js:280:320) at gc (react-dom.production.min.js:280:180) at ac (react-dom.production.min.js:271:88) fl @ react-dom.production.min.js:189 react-dom.production.min.js:127 Uncaught TypeError: Cannot read properties of undefined (reading '8022') at AnnouncePage.jsx:39:40 at AnnouncePage.jsx:166:36 at Array.forEach () at UW (AnnouncePage.jsx:165:22) at Ca (react-dom.production.min.js:167:137) at Cs (react-dom.production.min.js:290:337) at wc (react-dom.production.min.js:280:389) at bc (react-dom.production.min.js:280:320) at gc (react-dom.production.min.js:280:180) at ac (react-dom.production.min.js:271:88)

This is gold! THanks so much, and thanks for confirming that the cache clear fixed the issue. I've updated the Announce and Play By Play pages to handle the missing data, but I don't want to push that until later due to events running now.

--- a/src/components/AppUpdates.jsx
+++ b/src/components/AppUpdates.jsx
@@ -1,5 +1,12 @@
 export const appUpdates = [
   {
+    date: "March 28, 2025",
+    message: (
+      <ul>
+        <li>Fixing a bug that could cause a white screen while loading an event</li>
+      </ul>
+    ),
+  },{
     date: "March 25, 2025",
     message: (
       <ul>

--- a/src/pages/AnnouncePage.jsx
+++ b/src/pages/AnnouncePage.jsx
@@ -36,8 +36,8 @@ function AnnouncePage({ selectedEvent, selectedYear, teamList, rankings, communi
                     rankings?.ranks[_.findIndex(rankings?.ranks, { "teamNumber": team?.teamNumber })]
                 );
             team.rankStyle = rankHighlight(team?.rank, allianceCount || { "count": 8 });
-            team.alliance = alliances?.Lookup[`${team?.teamNumber}`]?.alliance || null;
-            team.allianceRole = alliances?.Lookup[`${team?.teamNumber}`]?.role || null;
+            team.alliance = alliances?.Lookup[`${team?.teamNumber}`] ? alliances?.Lookup[`${team?.teamNumber}`]?.alliance || null : null;
+            team.allianceRole = alliances?.Lookup[`${team?.teamNumber}`] ? alliances?.Lookup[`${team?.teamNumber}`]?.role || null : null;
 
             var teamDistrictRanks = _.filter(districtRankings?.districtRanks, { "teamNumber": team.teamNumber })[0] || null;
             team.districtRanking = teamDistrictRanks?.rank;

--- a/src/pages/PlayByPlayPage.jsx
+++ b/src/pages/PlayByPlayPage.jsx
@@ -31,8 +31,8 @@ function PlayByPlayPage({ selectedEvent, selectedYear, teamList, rankings, commu
                 communityUpdates[_.findIndex(communityUpdates, { "teamNumber": team?.teamNumber })]
             );
             team.rankStyle = rankHighlight(team?.rank, allianceCount || { "count": 8 });
-            team.alliance = alliances?.Lookup[`${team?.teamNumber}`]?.alliance || null;
-            team.allianceRole = alliances?.Lookup[`${team?.teamNumber}`]?.role || null;
+            team.alliance = alliances?.Lookup[`${team?.teamNumber}`] ? alliances?.Lookup[`${team?.teamNumber}`]?.alliance || null : null;
+            team.allianceRole = alliances?.Lookup[`${team?.teamNumber}`] ? alliances?.Lookup[`${team?.teamNumber}`]?.role || null : null;
 
             var teamDistrictRanks = _.filter(districtRankings?.districtRanks, { "teamNumber": team.teamNumber })[0] || null;
             team.districtRanking = teamDistrictRanks?.rank;
@@ -201,7 +201,7 @@ function PlayByPlayPage({ selectedEvent, selectedYear, teamList, rankings, commu
                 if (usePullDownToUpdate) {
                     getSchedule();
                 }
-                
+
             },
             preventScrollOnSwipe: true,
         }

